sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/IPrivateEndpointConnectionOperations.cs

```diff
@@ -119,7 +119,7 @@ public partial interface IPrivateEndpointConnectionOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<PrivateEndpointConnection,PrivateEndpointConnectionUpdateHeaders>> UpdateWithHttpMessagesAsync(string resourceGroupName, string accountName, string privateEndpointConnectionName, PrivateEndpointConnection parameters, string ifMatch = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<PrivateEndpointConnection>> UpdateWithHttpMessagesAsync(string resourceGroupName, string accountName, string privateEndpointConnectionName, PrivateEndpointConnection parameters, string ifMatch = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This is a breaking change as the return type of the function has changed. This can only be merged if there is an Approved-BreakingChange label on the Swagger PR.

Reply:
These were removed in error (the service does still return these headers). PR for updating the swagger here: https://github.com/Azure/azure-rest-api-specs/pull/12560

I'll regenerate the SDK and verify that this fixes the issue.

Reply:
Regenerated the SDK based on the swagger in that PR and it looks to have fixed the problem. Thanks for catching that!